doc/releases/changelog-dev.md

```diff
@@ -113,6 +113,9 @@
 * Empty initialization of `PauliVSpace` is permitted.
   [(#5675)](https://github.com/PennyLaneAI/pennylane/pull/5675)
 
+* `MultiControlledX` can now be decomposed even when no `work_wires` are provided. The implementation returns $\mathcal{O}(\text{len(control\_wires)}^2)$ operations, and is applicable for any multi controlled unitary gate. Single control unitary now includes the correct global phase. Single control `GlobalPhase` has now a decomposition, i.e. relative phase on control wire.
+  [(#5735)](https://github.com/PennyLaneAI/pennylane/pull/5735)
```

Comment:
These sound to me like 3 separate changelog entries.
```suggestion
* `MultiControlledX` can now be decomposed even when no `work_wires` are provided. The implementation returns $\mathcal{O}(\text{len(control\_wires)}^2)$ operations, and is applicable for any multi controlled unitary gate.
  [(#5735)](https://github.com/PennyLaneAI/pennylane/pull/5735)

* Single control unitary now includes the correct global phase.
  [(#5735)](https://github.com/PennyLaneAI/pennylane/pull/5735)

* Single control `GlobalPhase` has now a decomposition, i.e. relative phase on control wire.
  [(#5735)](https://github.com/PennyLaneAI/pennylane/pull/5735)
```

Reply:
Makes sense!